Source/Scene/Cesium3DTileStyle.js

```diff
@@ -425,6 +427,9 @@ define([
          * <p>
```

Comment:
Above - `pointSize` should mention that is only applicable to point features in a Vector tile or Point Cloud tile.

Reply:
> as a note - in the issue you mentioned pointSize is used by both but in the linked docs pointSize is only used for point features - for now i dont have the "This expression is only applicable to point features in a Vector tile" line for it - lmk if you want me to add it into the docs there as well

I didn't link to it, but `pointSize` is also used for Point Cloud tiles [here](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling#point-cloud)